src/drive/web/modules/drive/Toolbar/components/PortaledQueue.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
+import ReactDOM from 'react-dom'
+import { UploadQueue } from 'drive/web/modules/upload/UploadQueue'
+import { translate } from 'cozy-ui/transpiled/react'
+class PortaledQueue extends Component {
+  render() {
+    const { file, doneCount, successCount, t, onClear } = this.props
+
+    return ReactDOM.createPortal(
+      <UploadQueue
+        queue={file ? [file] : []}
+        doneCount={doneCount}
+        successCount={successCount}
+        t={t}
+        purgeQueue={onClear}
+      />,
+      document.querySelector('[role=application]')
+    )
+  }
+}
```

Comment:
This component's responsibility seems unclear to me. By its name, it seems like that it only does the portalling but it also

- changes the `purgeQueue` to `onClear`.
- changes the `queue` to `file`

I think we should limit/simplify the components responsibility.

- Why do we need to change the props ?
- I think UploadQueue should receive the `t` function by itself ?

Can you add a docstring explaining the responsibilities ?